plugin/v2/plugin.go

```diff
@@ -205,18 +213,53 @@ func (p *Plugin) GetTypes() []types.PluginInterfaceType {
 
 // InitSpec creates an OCI spec from the plugin's config.
 func (p *Plugin) InitSpec(s specs.Spec, libRoot string) (*specs.Spec, error) {
-	rootfs := filepath.Join(libRoot, p.PluginObj.ID, "rootfs")
+	p.Rootfs = filepath.Join(libRoot, p.PluginObj.ID, "rootfs")
 	s.Root = specs.Root{
-		Path:     rootfs,
+		Path:     p.Rootfs,
 		Readonly: false, // TODO: all plugins should be readonly? settable in manifest?
 	}
 
+	// TODO: the following should be only if granting device permissions
```

Comment:
Lets fix this todo by adding to Manifest's privilege section. Something like "REQUEST_DEVICE_CREATION".